CMakeLists.txt

```diff
@@ -654,19 +654,24 @@ install(TARGETS EXPORT snmallocConfig DESTINATION ${CMAKE_INSTALL_LIBDIR}
   PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/snmalloc)
 
 install(DIRECTORY src/snmalloc/aal DESTINATION include/snmalloc)
-install(DIRECTORY src/snmalloc/ds DESTINATION include/snmalloc)
-install(DIRECTORY src/snmalloc/override DESTINATION include/snmalloc)
 install(DIRECTORY src/snmalloc/backend DESTINATION include/snmalloc)
+install(DIRECTORY src/snmalloc/backend_helpers DESTINATION include/snmalloc)
```

Comment:
@mjp41 could you check the addition of installation targets?

Reply:
The cmake build target fails without these directories being added.

Reply:
This probably a good change, I think the directory structure changed after the install stuff was written, and it doesn't look like this was updated.